autoware_sensing_msgs/CMakeLists.txt

```diff
@@ -6,7 +6,8 @@ ament_auto_find_build_dependencies()
 
 set(msg_files
   "msg/GnssInsOrientation.msg"
-  "msg/GnssInsOrientationStamped.msg")
+  "msg/GnssInsOrientationStamped.msg"
```

Comment:
Could you put the parenthesis back up? To keep consistent.

Reply:
Addressed in [e3bbefa](https://github.com/autowarefoundation/autoware_msgs/pull/120/commits/e3bbefaa076b87379eb41edef0f9d68dbd5c6d90)